src/BenchmarkDotNet/Toolchains/CsProj/CsProjCoreToolchain.cs

```diff
@@ -26,8 +26,8 @@ public class CsProjCoreToolchain : Toolchain, IEquatable<CsProjCoreToolchain>
         [PublicAPI] public static readonly IToolchain NetCoreApp80 = From(NetCoreAppSettings.NetCoreApp80);
         [PublicAPI] public static readonly IToolchain NetCoreApp90 = From(NetCoreAppSettings.NetCoreApp90);
 
-        internal CsProjCoreToolchain(string name, IGenerator generator, IBuilder builder, IExecutor executor, string customDotNetCliPath)
-            : base(name, generator, builder, executor)
+        internal CsProjCoreToolchain(string name, IGenerator generator, IBuilder builder, IExecutor executor, string customDotNetCliPath, ISdkProvider sdkProvider)
```

Comment:
Why don't we merge the `customDotNetCliPath` into the `ISdkProvider`? They seem very related, and the validation likely would change depending on the custom path.

Reply:
> Why don't we merge the `customDotNetCliPath` into the `ISdkProvider`? They seem very related, and the validation likely would change depending on the custom path.

Will do. Should I do the same thing with `customDotNetCliPath` in `NativeAotToolchain` as well?

Reply:
Yes, please. All the toolchains that use it.